### PR TITLE
Made the mission statement page

### PR DESCRIPTION
**Pull request by arnavswami**
Made mission statement page. Small message and image of 1431 Everton Place on both sides. Let me know if you think that there should be any changes or if the styling should be different.

### CODE REVIEW

**Comment by arnavswami** [comment]
Image 2-16-24 at 11 18 PM

File: src/components/home/Mission.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      {/* Content container with 2/3 width */}
+      <div className="w-1/2 bg-neutral-300 flex flex-col justify-center items-center">
+        <div className="text-center">
+          <p
+            style={{
+              lineHeight: "2",
+              margin: "0",
+              marginTop: "10px",
+              marginRight: "20px",
+            }}
```
Comment (JaviChip) [inline src/components/home/Mission.jsx]: Dont use Style , thats CSS . Use TailwindCSS inline

File: src/components/home/Mission.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                fontSize: "38px",
+              }}
+            >
+              OUR MISSION: <br />
+            </span>
+          </p>
+          {/* Empty space column with 1/6 width */}
+          <div className="w-1/6"></div>
+          <p style={{ lineHeight: "2", margin: "0", marginRight: "10px" }}>
+            <span style={{ fontFamily: "Arial, sans-serif", fontSize: "22px" }}>
```
Comment (JaviChip) [inline src/components/home/Mission.jsx]: Dont use Style , use tailwindCSS

**Review by JaviChip — CHANGES_REQUESTED** [review]

File: src/components/home/Mission.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "next/image";
+import image1 from "../../../src/public/home/houseimage.svg";
+
+const Mission = () => {
+  return (
+    <div className="bg-neutral-300 flex w-full justify-center items-center">
+      {/* Left empty space column with 1/12 width */}
+      <div className="w-1/12"></div>
```
Comment (JaviChip) [inline src/components/home/Mission.jsx:9]: is this for spacing between the paragraph and image ?

File: src/components/home/Mission.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+      {/* Image container with 1/3 width */}
+      <div className="w-1/3 mt-6 mb-6">
+        <Image alt="" src={image1} className="ml-4 mr-auto" />
```
Comment (JaviChip) [inline src/components/home/Mission.jsx:13]: Add an alt text for eslint purposes

File: src/components/home/Mission.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      <div className="w-1/6"></div>
+
+      {/* Content container with 2/3 width */}
+      <div className="w-1/2 bg-neutral-300 flex flex-col justify-center items-center">
+        <div className="text-center">
+          <p className="text-blue-500 font-bold font-serif text-2xl md:text-3xl lg:text-4xl leading-8 m-0 mt-10 mr-20">
+            OUR MISSION: <br />
+          </p>
+          {/* Empty space column with 1/6 width */}
+          <div className="w-1/6"></div>
```
Comment (JaviChip) [inline src/components/home/Mission.jsx:26]: what does this do ?

**Review by JaviChip — APPROVED** [review]

**Pull request merged**